dev-libs/jansson/jansson-2.14.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal
+
+DESCRIPTION="C library for encoding, decoding and manipulating JSON data"
+HOMEPAGE="https://www.digip.org/jansson/"
+SRC_URI="https://github.com/akheron/jansson/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="2/4"
```

Comment:
We need to add a [subslot](https://devmanual.gentoo.org/general-concepts/slotting/index.html#sub-slots) to force rebuilding, but no need to change the main slot (0->2).